graphql/src/datasources/es.ts

```diff
@@ -138,20 +162,47 @@ class ElasticSearchAPI extends RESTDataSource {
       };
     }
 
-    let filters: { [key: string]: string }[] = [];
+    let divisionIds = administrativeDivisionIds ?? [];
     if (administrativeDivisionId) {
```

Comment:
Is this left here to support backwards compatibility or is this somehow different to just passing one id in the array?